k8s/celery/turbinia-api-server.yaml

```diff
@@ -18,16 +18,17 @@ spec:
         app: turbinia-api-server
     spec:
       serviceAccountName: turbinia
-      initContainers:
-        - name: init-filestore
-          image: busybox:1.28
-          command: ['sh', '-c', 'chmod go+w /mnt/turbiniavolume']
-          volumeMounts:
-            - mountPath: "/mnt/turbiniavolume"
-              name: turbiniavolume
+      automountServiceAccountToken: false
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+        runAsUser: 999
+        fsGroup: 999
+        fsGroupChangePolicy: "OnRootMismatch"
       containers:
         - name: api
-          image: us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-api-server:latest
+          image: us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-api-server-dev:latest
```

Comment:
merge conflict?

Reply:
no I changed it myself during testing, but will change back as forgot to change it back when submitting for review